src/Zend/View/Helper/HeadLink.php

```diff
@@ -365,6 +365,7 @@ public function createDataStylesheet(array $args)
         $media                 = 'screen';
         $conditionalStylesheet = false;
         $href                  = array_shift($args);
+        $extras                = array();
```

Comment:
Just pre-defining this `$extras` variable (in both cases) so it's not undefined when it gets passed to `compact`, which will throw a notice in php 7.3. https://secure.php.net/compact